### ISSUE TITLE
Low contrast text is difficult to read and does not meet accessibility standards

### ISSUE THREAD

**Issue by ashanan** [politics-rewired/Spoke#1065]
The org I volunteer with uses spoke to run text banks, and I regularly see people struggle with the low contrast grey text on grey backgrounds.  The "current question" dropdown is the hardest for new folks, especially if they have any kind of visual impairment.  Using firefox's built in accessibility tools I was able to determine that the text in question has a contrast ratio well below the WCAG recommendation (see screenshot below).

![Screen Shot 2022-02-08 at 4 23 18 PM](https://user-images.githubusercontent.com/11887/153086882-238e4561-9aa8-40c5-a6f2-03b24e3ef0dd.png)

**Comment by ashanan** [comment]
Additional info: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast

**Comment by ashanan** [comment]
Thanks for labeling this ticket @bchrobot!  I think it would also be appropriate to label this as a bug given that the site is unusable for folks with visual impairments.

**Comment by ninabaldwin** [comment]
Another user has requested this feature be added.

**Comment by ashanan** [comment]
I want to start off by thanking @bchrobot for both fixing this in the above PR and for correctly labeling it as a bug fix.

@ninabaldwin having a usable site is not a feature.  Accessibility is not a feature -- it is table stakes for treating users with disabilities equitably (never mind that it's also basically a requirement of the ADA).

**Comment by ashanan** [comment]
@bchrobot thanks for fixing this!  What do I need to do to ensure that my org's instance is updated to the version that has this fix?

**Comment by bchrobot** [comment]
> What do I need to do to ensure that my org's instance is updated to the version that has this fix?

This fix has been included in the 4.20.0 release. That has just gone out to our beta release channel and, assuming no regressions are found, will be released to all clients on Monday.
